browser-test/src/support/index.ts

```diff
@@ -205,3 +212,19 @@ export const validateAccessibility = async (page: Page) => {
 
   expect(results).toHaveNoA11yViolations()
 }
+
+export const validateScreenshot = async (
+  page: Page,
+  pageScreenshotOptions?: PageScreenshotOptions,
+  matchImageSnapshotOptions?: MatchImageSnapshotOptions,
+) => {
+  expect(
+    await page.screenshot({
+      ...pageScreenshotOptions,
+    }),
+  ).toMatchImageSnapshot({
+    failureThreshold: 0.03,
+    failureThresholdType: 'percent',
+    ...matchImageSnapshotOptions,
```

Comment:
Can we group screenshots by test files? Currently they all stored in one giant directory which might become harder to work with the more screenshots we get. Putting them to per-test-file directories would help a bit.

Additionally we should anticipate mobile tests as well. Currently all screenshots are taken on desktop. But civiform needs to be well-optimized for mobile and that will be important part of the browser testing. We could add "desktop/mobile" suffix to screenshots from the start or put them in separate "desktop"/"mobile" directories so that later we won't have to move a lot of files.

Reply:
Yes, you can specify a file path parameter for saving the images. They're currently already grouped according to the test name, but we can also put them in directories as well.